CommunityBugFixCollection/SteamVrFocusResolutionScale.cs

```diff
@@ -0,0 +1,29 @@
+﻿using HarmonyLib;
+using MonkeyLoader.Resonite;
+using System.Collections.Generic;
+
+namespace CommunityBugFixCollection
+{
+    [HarmonyPatchCategory(nameof(SteamVrFocusResolutionScale))]
+    [HarmonyPatch("Valve.VR.SteamVR_Render", "OnInputFocus")]
+    internal sealed class SteamVrFocusResolutionScale : ResoniteMonkey<SteamVrFocusResolutionScale>
+    {
+        public override IEnumerable<string> Authors => Contributors.Goat;
+        
+        private static bool _lastInputFocus = false;
```

Comment:
Are we sure that there will only ever be one instance of the SteamVR_Render class? Otherwise it may be prudent to use a dictionary anyways.